packages/web-components/src/components/ic-navigation-group/ic-navigation-group.css

```diff
@@ -166,6 +166,7 @@
   transition: var(--ic-easing-transition-slow);
   overflow: hidden;
 }
+
```

Comment:
Is this a necessary change?

Reply:
This is just to improve the files readability. New block of CSS so have a empty line to separate it.